docker-compose.dev.yml

```
@@ -0,0 +1,15 @@
+version: "3.8"
+services:
+  web:
+    build:
+      context: .
+      target: development
+    ports:
+      - "127.0.0.1:5000:5000"
```

Yaml treats this as a string anyway
```suggestion
      - 127.0.0.1:5000:5000
```